exchanges/poloniex/poloniex.go

```diff
@@ -3,11 +3,12 @@ package poloniex
 import (
```

Comment:
Please open `exchange_wrapper_standards.go` and remove `poloniex` from `unsupportedExchangeNames` map.

You may wish to add `websocket.ErrWebsocketNotEnabled` to acceptable errors